server/rest/submission.py

```diff
@@ -310,6 +311,49 @@ def postSubmission(self, phase, folder, params):
 
         return self._filterScore(phase, submission, user)
 
+    @access.admin
+    @loadmodel(model='submission', plugin='covalic')
+    @filtermodel(model='submission', plugin='covalic')
+    @describeRoute(
+        Description('Overwrite metadata for a submission.')
+        .param('id', 'The ID of the submission.', paramType='path')
+        .param('title', 'Title for the submission', required=False)
+        .param('date', 'The date of the submission.', required=False)
+        .param('organization', 'Organization associated with the submission.',
+               required=False)
+        .param('organizationUrl', 'URL for organization associated with the submission.',
+               required=False)
+        .param('documentationUrl', 'URL of documentation associated with the submission.',
+               required=False)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Site admin access is required.', 403)
+    )
+    def updateSubmission(self, submission, params):
+        title = self._getStrippedParam(params, 'title')
+        if title is not None:
+            submission['title'] = title
+        created = self._getStrippedParam(params, 'date')
+        if created is not None:
+            submission['created'] = created
+        # Since only admins may use this endpoint, skip the normal checks for
+        # 'enableOrganization', etc.
+        organization = self._getStrippedParam(params, 'organization')
+        if organization is not None:
+            submission['organization'] = organization
+        organizationUrl = self._getStrippedParam(params, 'organizationUrl')
+        if organizationUrl is not None:
+            submission['organizationUrl'] = organizationUrl
+        documentationUrl = self._getStrippedParam(params, 'documentationUrl')
+        if documentationUrl is not None:
+            submission['documentationUrl'] = documentationUrl
+
+        submission = self.model('submission', 'covalic').save(submission)
+
+        user = self.getCurrentUser()
+        phase = self.model('phase', 'covalic').load(
```

Comment:
I can't put it on the decorator, since `Submission` is not an `AccessControlledModel`.

I think the best solution is for `Submission` to use `AccessControlMixin` against it's containing `Phase`, but that's way outside of scope here.

I'll move the `Phase.load` to run before `Submission.save`, and require `level=WRITE` on the `Phase`.